6. Database/models/product.js

```diff
@@ -0,0 +1,34 @@
+const products = [];
+let id =1;
+
+const db = require('../utils/database');
+
+module.exports = class product {
+    constructor(title,imageUrl,price,description) {
+        this.title = title;
+        this.imageUrl = imageUrl;
+        this.price = price;
+        this.description = description;
+    }
+
+    save() {
+        this.id = product.id();
+        return db.execute('INSERT INTO Products VALUES(?,?,?,?,?)',[this.id,this.title,this.imageUrl,this.price,this.description]);
+    }
+
+    static fetchAll() {
+        return db.execute('SELECT * FROM Products');
+    }
+
+    static delete(index) {
+        products.splice(index,1);
```

Comment:
why aren't you using a delete query to delete products? any particular reason removes it from the array.

Reply:
I've forgot to change that.

that was old code, when i was storing the products in an array, and to delete the product i was removing that element from a productArray, but now i'm storing products in database, so to delete the product ,
here is the method:

static delete(id) {
        return db.execute('DELETE FROM Products WHERE id = ? ',[id]);
}